### PR TITLE
Nfc tools

### PR DESCRIPTION
**Pull request by thypon**
#1369 here the two most used nfc-tools lib/apps

### CODE REVIEW

File: srcpkgs/libnfc/template [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+hostmakedepends="cmake pkg-config"
+makedepends="libusb-devel libusb-compat-devel"
+maintainer="Andrea Brancaleoni <miwaxe@gmail.com>"
+license="GPL-3"
+homepage="http://nfc-tools.org/"
+distfiles="https://github.com/nfc-tools/libnfc/archive/libnfc-${version}.tar.gz"
+checksum=30de35b4f1af3f57dab40d91ffb2275664a35859ff2b014ba7b226aa3f5465f5
+
+pre_configure() {
+	sed -i "s|MODE=\"0664\", GROUP=\"plugdev\"|TAG+=\"uaccess\"|g" contrib/udev/42-pn53x.rules
```
Comment (xtraeme) [inline srcpkgs/libnfc/template]: We don't have a `plugdev` group.

File: srcpkgs/mfoc/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+makedepends="libnfc-devel"
+short_desc="Mifare Classic Offline Cracker"
+maintainer="Andrea Brancaleoni <miwaxe@gmail.com>"
+license="GPL-3"
+homepage="http://nfc-tools.org"
+distfiles="https://github.com/nfc-tools/mfoc/archive/${pkgname}-${version}.tar.gz"
+checksum=2dfd8ffa4a8b357807680d190a91c8cf3db54b4211a781edc1108af401dbaad7
+
+pre_configure() {
+	aclocal
```
Comment (xtraeme) [inline srcpkgs/mfoc/template]: You can replace those commands with `autoreconf -fi`

File: srcpkgs/mfoc/template [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'mfoc'
+pkgname=mfoc
+version=0.10.7
+revision=1
+wrksrc="${pkgname}-${pkgname}-${version}"
+build_style=gnu-configure
+hostmakedepends="autoconf automake pkg-config"
```
Comment (xtraeme) [inline srcpkgs/mfoc/template]: `autoconf` is a dependency of `automake`, so that it's redundant.

File: srcpkgs/libnfc/template (hunk 1 above)
Comment (thypon) [inline srcpkgs/libnfc/template]: In this sed i removed plugdev and add tag uaccess

File: srcpkgs/mfoc/template (hunk 2 above)
Comment (thypon) [inline srcpkgs/mfoc/template]: check

File: srcpkgs/mfoc/template (hunk 3 above)
Comment (thypon) [inline srcpkgs/mfoc/template]: check

File: srcpkgs/libnfc/template (hunk 1 above)
Comment (xtraeme) [inline srcpkgs/libnfc/template]: You are right, I saw this backwards :-)

**Comment by thypon** [comment]
Now it's okay

**Pull request merged**